packages/design-tokens/package.json

```diff
@@ -20,7 +20,7 @@
   "scripts": {
     "build": "run-s build:dict build:cjs types",
     "build:dict": "node ./build-dictionary.js",
-    "build:cjs": "webpack --mode production --config=webpack/prod.js",
```

Comment:
`-mode production` was redundant as it's specified in the configuration (and is the default also).